### ISSUE TITLE
Reproducible recipes

### ISSUE THREAD

**Issue by fminkin** [facebookresearch/wav2letter#101]
Hello!

I wonder if your recipe in librispeech is the one achieving WER published in paper? If so, can you tell the time it'll took the model to converge on 8 V100?

**Comment by vineelpratap** [comment]
Hi,
 Yes, the architecture in the recipe is the one that achieves the best WER as published in paper for Librispeech. Note that we have done careful tuning of learning rate to achieve best WER and used distributed training on 4 GPUs with a batchsize of 4 per GPU.

 With 8 V100 GPUs , my guess would be that it will take around 1-2 weeks to get best WER reported in paper. However, the model will converge to a good LER (< 5% LER on clean speech) must faster (1-2 days).

**Comment by fminkin** [comment]
Thanks. 
Could you share some experiments on some learning rate strategies and their effect on your network? Simply increasing proportionaly to batch size growth works fine?

**Comment by vineelpratap** [comment]
Hi,
 Note that we already scale learning rate with batchsize - https://github.com/facebookresearch/wav2letter/blob/master/Train.cpp#L365

I would recommend doing a grid search around the optimal learning rate parameters mentioned in the config and choose the best. (For the sake of completeness) Once the loss has stopped decreasing and you may want to decrease the learning rate further to get best results.
